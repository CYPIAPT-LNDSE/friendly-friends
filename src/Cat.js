import React from 'react';

const Cat = () => {
  return (
    <img className="cat rotating" src="./cat.png" />
  );
}

export default Cat;
