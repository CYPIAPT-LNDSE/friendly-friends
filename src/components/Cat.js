import React from "react";

const Cat = ({ initiateConversation, isBouncing }) => {
  const classes = `cat ${isBouncing ? "backflip" : ""}`;
  return (
    <img onClick={() => initiateConversation(12)} alt="friendly cat" className={classes} src="./cat.png" />
  );
}

export default Cat;
