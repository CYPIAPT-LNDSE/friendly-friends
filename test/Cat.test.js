import React from 'react';
import renderer from 'react-test-renderer';
import Cat from '../src/components/Cat.js';

test('Cat renders correctly', () => {
  const tree = renderer.create(<Cat/>).toJSON();
  expect(tree).toMatchSnapshot();
});
