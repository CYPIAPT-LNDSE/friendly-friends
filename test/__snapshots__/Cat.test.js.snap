exports[`test Cat renders correctly 1`] = `
<img
  alt="friendly cat"
  className="cat"
  src="./cat.png" />
`;
